src/sortStudents.ts

```diff
@@ -1,16 +1,50 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageAge'
 }
 
-// create SortOrder type
-export type SortOrder;
+export enum SortOrder {
+  asc = 'asc',
+  desc = 'desc',
```

Comment:
```suggestion
  Asc = 'asc',
  Desc = 'desc',
```
capitalize for enum names